TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -2,7 +2,7 @@ const Parser = require( './Parser' );
 const benchmarkDelimiterRegex = /\*\*\*\*\*\*\*\*\*\* START OF NEW TESTCI BENCHMARK JOB \*\*\*\*\*\*\*\*\*\*[\r\n]/;
 const benchmarkNameRegex = /Benchmark Name: (.*) Benchmark Variant: .*[\r\n]/;
 const benchmarkVariantRegex = /Benchmark Name: .* Benchmark Variant: (.*)[\r\n]/;
-const productResourceRegex = /Product Resource: (.*)[\r\n]/;
+const productResourceRegex = /SDK_RESOURCE=(.*)[\r\n]/;
```

Comment:
We shouldn't change the regex for this since PerfNext or any tool using Benchmark Parser follows that convention and doesn't have `SDK_RESOURCE`, which is only for TKG builds.